core-bundle/src/Image/Studio/FigureBuilder.php

```diff
@@ -773,18 +773,28 @@ private function onDefineLightboxResult(Figure $result): LightboxResult|null
                 return [null, $target];
             }
 
-            if (Path::isAbsolute($target)) {
-                $filePath = Path::canonicalize($target);
-            } else {
-                // URL relative to the project directory
-                $filePath = Path::makeAbsolute(urldecode($target), $this->projectDir);
+            $filePath = urldecode($target);
+
+            // Check if target is an absolute filesystem path to an existing resource
+            if (Path::isAbsolute($filePath) && is_file($filePath)) {
+                return [Path::canonicalize($filePath), null];
+            }
```

Comment:
This seems wrong to me. If `$target` is an absolute path it should not be URL-encoded here and decoding it would potentially break it.

```suggestion
            // Check if target is an absolute filesystem path to an existing resource
            if (Path::isAbsolute($target) && is_file($target)) {
                return [Path::canonicalize($target), null];
            }

            $filePath = urldecode($target);
```